services/github-service/src/test/java/org/kontinuity/catapult/service/github/impl/kohsuke/GitHubServiceIT.java

```diff
@@ -75,11 +75,16 @@ public void cannotForkNonexistentRepo(){
    
     @Test
     public void createGithubWebHook() throws Exception{
+    	final URL webhookUrl = new URL("https://10.1.2.2");
+    	
     	final GitHubRepository targetRepo = gitHubService.fork(NAME_GITHUB_SOURCE_REPO);
-    	gitHubService.createWebHook(
-    			targetRepo.getFullName(),
-    			"https://10.1.2.2",						// TODO Webhook URL
-    			new KohsukeGithubWebhookEvent("ALL"));
+		GitHubWebhook webhook = gitHubService.createWebhook(
+    			targetRepo,
+    			webhookUrl,
+    			new KohsukeGitHubWebhookEvent("ALL"));
```

Comment:
Correct, so I'm converting GitHubWebhookEvent to an enum and removing KohsukeGitHubWebhookEventImpl.